car-data-app/pages/index.js

```diff
@@ -0,0 +1,128 @@
+            placeholder="Model..."
+            onChange={(e) => setModel(e.target.value)}
+          />
+          <button
+            className="outline-none border border-active font-bold font-raleway ml-4 px-12 py-2 rounded-sm bg-secondary text-bc transition duration-300 hover:bg-bc hover:text-primary md:ml-0 md:mt-4"
+            onClick={(e) => fetchCarDetails(e)}
+          >
+            Search
+          </button>
+        </form>
+
+        <div className="flex flex-col text-primary text-sm text-raleway mt-12 w-4/6 h-4/5  md:flex-col md:w-4/6 md:h-full md:mb-12">
+          <table className="bg-white w-full text-primary mb-8 md:text-sm md:mx-2">
+            <thead className="font-raleway uppercase tracking-wide text-secondary">
+              <tr>
+                <th className="border border-x-0 text-left px-4 py-4">
+                  <span className="text-danger">Year</span>
+                </th>
+                <th className="border border-x-0 text-left px-4 py-4">
+                  <span className="text-danger">Type</span>
+                </th>
+                <th className="border border-x-0 text-left px-4 py-4">
+                  <span className="text-danger">Make</span>
+                </th>
+                <th className="border border-x-0 text-left px-4 py-4">
+                  <span className="text-danger">Model</span>
+                </th>
+              </tr>
+            </thead>
+            <tbody className="text-primary">
+              {res.map((info) => (
+                <tr>
+                  <th className="border border-x-0 border-secondary text-left px-4 py-4">
+                    <span className="text-danger">{info.year}</span>
+                  </th>
+                  <th className="border border-x-0 border-secondary text-left px-4 py-4">
+                    <span className="text-danger">{info.type}</span>
+                  </th>
+                  <th className="border border-x-0 border-secondary text-left px-4 py-4">
+                    <span className="text-danger">{info.make}</span>
+                  </th>
+                  <th className="border border-x-0 border-secondary text-left px-4 py-4">
+                    <span className="text-danger">{info.model}</span>
+                  </th>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </div>
+      </div>
+      <div className="flex flex-col mt-10 justify-end h-36 md:h-24">
+        <p className="block mb-10 text-center text-secondary text-xs">
+          Made by RapidAPI DevRel Team -{" "}
+          <a
+            className="hover:text-active"
+            href="https://github.com/RapidAPI/DevRel-Examples-External"
+          >
+            See more examples like this
+          </a>
+        </p>
+      </div>
+    </div>
+  );
+}
+
+export async function getServerSideProps() {
+  const res = await axios.get("http://localhost:3000/api/info");
```

Comment:
How is this working on the production?

Reply:
This is the example code for the article, not the one in production. If you remember that when I started building these apps, we ran into an issue of a maximum of three deployments per repository in Vercel's free package. 

So, I presented that we create a separate private repository in our GitHub account for each app and deploy that repo instead.